journal/202110.md

```diff
@@ -0,0 +1,450 @@
+# {DRAFT} Decred Journal – October 2021
+
+![{alt text}](../img/{file.ext} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+Highlights for October: {
+
+- Bitcoin SPV and BTC registration fees merged in DCRDEX
+- maybe summarize that things are shaping up for the next release
+
+}
+
+Contents: { sync }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Ecosystem](#ecosystem)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Discussions](#discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+_dcrd is a full node implementation that powers Decred's peer-to-peer network around the world._
+
+Merged changes:
+
+- updating of indexes (transaction, address, exists address) has been made [asynchronous](https://github.com/decred/dcrd/pull/2219) to speed up the critical path of block validation and connection code { any other big picture or fun stuff to mention for it? and does it help multipeer downloads? }
```

Comment:
It allows for faster vote propagation and helps pave the way for other optimizations, sync models, and ultimately nicer data corruption recovery mechanisms.

Reply:
Nice explainer! Added.